index/server/main.go

```diff
@@ -197,11 +198,35 @@ func main() {
 	router.HEAD("/v2/*proxyPath", ociServerProxy)
 	router.GET("/v2/*proxyPath", ociServerProxy)
 
+	router.GET("/viewer", serveUI)
+	router.GET("/viewer/*proxyPath", serveUI)
+
+	// Serve static content for stacks
 	router.Static("/stacks", stacksPath)
 
 	router.Run(":8080")
 }
 
+// serveDevfileIndex serves the index.json file located in the container at `serveDevfileIndex`
```

Comment:
We need to update the function comment for `serveRootEndpoint`

Reply:
Thanks. Updated.